Readme.md

```diff
@@ -2,6 +2,10 @@
 
 # REST client for Deon Digital's API
 
+### [Unreleased]
+
+* Correct the `Duration` validation logic to match the intended format. Now supports `PnDTnHnMnS` instead of `PnYnMnDTnHnMnS`.
```

Comment:
Maybe mention something like `as specified in ISO XXXX`./

Reply:
Updated it